src/zmalloc.h

```diff
@@ -168,6 +168,17 @@ size_t zmalloc_usable_size(void *ptr);
  * obtained from z[*]_usable() family functions, there is no need for this step. */
 #define zmalloc_usable_size(p) zmalloc_size(p)
 
+#ifdef HAVE_MALLOC_SIZE
+#define PREFIX_SIZE 0
```

Comment:
If we expose this in the zmalloc api, then we should prefix it like ZMALLOC_PREFIX_SIZE, but I don't think we nees to expose it.

We already have zmalloc_size vs zmalloc_usable_size.